integration-tests/tests/fixtures/base_config.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+import pytest
+from tests.utils.config import BaseConfig
+from tests.utils.utils import get_env_var
+
+
+@pytest.fixture(scope="session")
+def base_config() -> BaseConfig:
+    clp_build_dir = Path(get_env_var("CLP_BUILD_DIR")).resolve()
+    clp_package_dir = Path(get_env_var("CLP_PACKAGE_DIR")).resolve()
+
+    base_config = BaseConfig(
+        clp_bin_dir=clp_package_dir / "bin",
+        clp_package_dir=clp_package_dir,
+        clp_sbin_dir=clp_package_dir / "sbin",
+        test_output_dir=clp_build_dir / "var" / "logs" / "pytest",
+        uncompressed_logs_dir=clp_build_dir / "var" / "data" / "pytest" / "downloads",
```

Comment:
technically, we don't have to put the test output to under var/.
how about we just create a "integration-test" directory under clp_build_dir?